OpenRA.Mods.Common/Traits/Health.cs

```diff
@@ -159,6 +159,9 @@ public void Resurrect(Actor self, Actor repairer)
 
 		public void InflictDamage(Actor self, Actor attacker, Damage damage, bool ignoreModifiers)
 		{
+			if (self.Owner.PlayerActor.Trait<DeveloperMode>().Immortal)
+				return;
```

Comment:
I don't think this is the play. With this you make your every unit immortal, not a specific one you selected

Reply:
To be honest - I wanted to make everything immortal, not only a single unit.

Reply:
Usually testing requires killing

Reply:
You still can kill others to your hearts content!

Reply:
In vast majority of cases I need to kill my own troops. Forcing to test things with 2 clients is sub-par

Reply:
So it would be better if the immortality would be applied to selected actors?

Reply:
Imo yes.